apps/schematic/api/schematic_api/openapi/openapi.yaml

```diff
@@ -1082,6 +1082,104 @@ paths:
       tags:
       - Schema
       x-openapi-router-controller: schematic_api.controllers.schema_controller
+  /generateGoogleSheetManifests:
+    get:
+      description: Generates a list of google sheet links
+      operationId: generate_google_sheet_manifests
+      parameters:
+      - description: The URL of a schema in jsonld form
```

Comment:
After the schema refactor, the schema url no longer has to be in jsonld form. After you update to use schematic `v24.1.1`,  the schema url could also be a link to the data model csv.